design/one-pager-terrajet-metadata-extraction.md

```diff
@@ -0,0 +1,272 @@
+# Metadata Extraction from Terraform Registry for Terrajet-based providers
```

Comment:
```suggestion
# Metadata Extraction from Terraform Registry in Terrajet
```

Since we're removing the difference between Terrajet vs SDK calls at repository level, I think removing `Terrajet-based provider` or `Terrajet-based repository` terms would be more future-proof for readers.